.circleci/config.yml

```diff
@@ -58,7 +58,7 @@ jobs:
       - run:
           command: |
             . venv/bin/activate
-            python -m  flake8 --ignore=E731,E501,E722,W503,E126,F401,E203,F403 gradio test
+            python -m  flake8 --ignore=E731,E501,E722,W503,E126,F401,E203,F403,F541 gradio test
```

Comment:
why did we add this ignore rule?

Reply:
Cant remember it now, but was probably smt needed and cannot be bypassed.